cfme/containers/project.py

```diff
@@ -135,10 +133,15 @@ def resetter(self):
             self.view.toolbar.view_selector.select("Dashboard View")
 
 
-@navigator.register(Project, 'EditTags')
-class EditTags(CFMENavigateStep):
+@navigator.register(Project, 'EditTagsFromDetails')
+class EditTagsFromDetails(CFMENavigateStep):
     VIEW = TagPageView
     prerequisite = NavigateToSibling('Details')
 
     def step(self):
         self.prerequisite_view.toolbar.policy.item_select('Edit Tags')
+
+
+@navigator.register(Project, 'EditTagsFromDashboard')
+class EditTagsFromDashboard(EditTagsFromDetails):
```

Comment:
REQUIRED: I suppose this is typo. please check this out again

Reply:
@psav, could you remind me if this is ok ?  ^^
Afaik, we had some issues with step inheritance before.

Reply:
I don't see a typo here. What are you seeing?

Reply:
you inherit one step from another one. I thought this is typo. I got later that you did this intentionally.
let's wait for @psav reply. I'd like to have a confirmation.

Reply:
Hmmmm - I'm not sure it's a supported config, but perhaps it could be, @izapolsk can you remember the issues we faced?

Reply:
There's not much code that gets repeated for these navigation destinations, and I'm a bit hesitant to connect them in this way, because of the potential changes to navigation that would then impact multiple pages.

Reply:
I will change it